manifests/components/grafana.jsonnet

```diff
@@ -68,15 +69,14 @@ local GRAFANA_DATASOURCES_CONFIG = "/opt/bitnami/grafana/conf/provisioning/datas
   },
 
   // Generates YAML configuration under provisioning/datasources/
-  datasources: kube.ConfigMap($.p + "grafana-prometheus") + $.metadata {
+  datasources: kube.ConfigMap($.p + "grafana-prometheus-datasource") + $.metadata {
     local this = self,
     datasources:: {
       // Built-in datasource for BKPR's Prometheus
       "BKPR Prometheus": {
         type: "prometheus",
-        access: "server",
-        InsecureSkipVerify: "true",
-        orgId: 1,
+        access: "proxy",
+        isDefault: true,
         url: "http://%s:9090/" % $.prometheus.host,
```

Comment:
I know it isn't part of this change, but: `url: $.prometheus.http_url` ?

Reply:
(I'm also ok with merging this as-is)

Reply:
I think `prometheus.http_url` equals to the ingress URL of Prometheus, and now that Prometheus is configured behind the oAuth proxy it can affect to the queries, that's why is configured as the internal URL (`prometheus.kubeprod.svc`)

Reply:
@gus is correct and $.prometheus.http_url generates `url: http://prometheus.kubeprod.svc:9090/`